entity/templates/src/main/webapp/app/entities/_entity.js

```diff
@@ -90,7 +90,7 @@ angular.module('<%=angularAppName%>')
             })
             .state('<%= entityInstance %>-management.edit', {
                 parent: '<%= entityInstance %>-management',
-                url: '/{id:int}/edit',
+                url: '/{id:<% if (databaseType == 'sql') { %>int<% } else if (databaseType == 'mongodb') { %>[0-9a-fA-F]{24}<% } else if (databaseType == 'cassandra') { %>[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}<% } %>}/edit',
```

Comment:
@andidev may I suggest doing the conditional in the entity/index.js and assign to a var `this.idType` and then just assign it here?  It would be more efficient and less ugly.

Infact we should try to move more logic from templates to main js